src/coreclr/tools/aot/ILCompiler/repro/repro.csproj

```diff
@@ -9,7 +9,6 @@
     <RuntimeIdentifiers>linux-x64;win-x64;osx-x64;freebsd-x64</RuntimeIdentifiers>
     <Configurations>Debug;Release;Checked</Configurations>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
Are we OK with the default setting changing here?

Reply:
I made that change (#63910) based on Stephen's comment in https://github.com/dotnet/runtime/pull/63777#issuecomment-1013206253. But it never actually worked because we specify WarnAsError on the command line in build.cmd and that one overwrites everything.

That happened in a period when a lot of other things were getting worse in my life so I didn't ever follow up because getting fewer build breaks in that project wouldn't make a significant difference to my quality of life anymore.